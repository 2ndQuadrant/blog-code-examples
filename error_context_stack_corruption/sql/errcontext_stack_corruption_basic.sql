\set client_min_messages notice
\set VERBOSITY verbose
SELECT errcontext_stack_corruption.my_func(true, 'simple');
SELECT errcontext_stack_corruption.my_func(false, 'simple');
