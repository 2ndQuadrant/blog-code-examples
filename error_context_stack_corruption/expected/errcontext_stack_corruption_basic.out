\set client_min_messages notice
SELECT errcontext_stack_corruption.my_func(true, 'simple');
INFO:  did the thing
INFO:  after return from requested call
 my_func 
---------
 
(1 row)

SELECT errcontext_stack_corruption.my_func(false, 'simple');
WARNING:  not doing it!
INFO:  after return from requested call
 my_func 
---------
 
(1 row)

